apps/InfiniteRedAI/app/screens/DocumentScannerScreen.tsx

```diff
@@ -18,13 +13,6 @@ interface DocumentScannerScreenProps
 export const DocumentScannerScreen: FC<DocumentScannerScreenProps> = observer(
   function DocumentScannerScreen() {
     const navigation = useTypedNavigation<"ImageLabeling">()
```

Comment:
Can you update this other missed part, `useTypedNavigation<"ImageLabeling">()` => `useTypedNavigation<"DocumentScanner">()?

Reply:
good catch!